features/doctrine/date_filter.feature

```diff
@@ -29,13 +29,16 @@ Feature: Date filter on collections
                   {"pattern": "^/dummies/29$"}
                 ]
               }
-            },
-            "hydra:view": {
-              "@id": {"pattern": "^/dummies\\?dummyDate\\[after\\]=2015-04-28$"},
-              "@type": {"pattern": "^hydra:PartialCollectionView$"}
             }
           },
           "maxItems": 2
+        },
+        "hydra:view": {
```

Comment:
Anyway, I'm curious about these changes. If the `hydra:view` was at the wrong level, does it mean that the previous JSON Schema was wrong / not checking the correct things?